packages/core/http/core-http-router-server-internal/src/router.ts

```diff
@@ -24,12 +24,17 @@ import type {
   RequestHandler,
 } from '@kbn/core-http-server';
 import { validBodyOutput } from '@kbn/core-http-server';
+import { performance } from 'perf_hooks';
+import apm from 'elastic-apm-node';
 import { CoreKibanaRequest } from './request';
 import { kibanaResponseFactory } from './response';
 import { HapiResponseAdapter } from './response_adapter';
 import { wrapErrors } from './error_wrapper';
 import { RouteValidator } from './validator';
 
+const THRESHOLD_ELU = 0.15;
+const THRESHOLD_ELA = 250;
```

Comment:
We'll want to have this configurable, but it can be done in a follow-up

Reply:
Wow! Installing any of the sample datasets exceeds these 😨 

```
[2023-04-10T16:31:44.369+02:00][WARN ][http] Event loop utilization for /api/sample_data/ecommerce exceeded threshold of 250ms and 15%: 1131ms out of 2529ms (45%)
```

While the Ops Metrics show a very acceptable Even Loop Delay:

```
[2023-04-10T16:31:44.890+02:00][DEBUG][metrics.ops] memory: 493.2MB uptime: 0:09:30 load: [3.95,5.38,6.27] mean delay: 11.125 delay histogram: { 50: 11.035; 95: 11.657; 99: 14.664 } utilization: 0.24221
```